tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/NamedTuplesSuite.scala

```diff
@@ -31,6 +31,41 @@ class NamedTuplesSuite extends BaseDottySuite {
     )))
 
   }
+  test("simple-named-type-assign") {
+    runTestAssert[Stat]("type T = (a: Int, b: String)")(Defn.Type(
+      Nil,
+      Type.Name("T"),
+      Type.ParamClause(Nil),
+      Type.Tuple(List(
+        Type.TypedParam(Type.Name("a"), Type.Name("Int"), Nil),
+        Type.TypedParam(Type.Name("b"), Type.Name("String"), Nil)
+      )),
+      Type.Bounds(None, None)
+    ))
+
+  }
+
+  test("named-tuple-summon") {
+    val code = """|val y = summon[Tuple2[Int, String] <:< (x: Int, y: String)]
```

Comment:
is `<:<` another type of bound? if so, should it be defined as an own token?

and should `(x: Int, y: String)` really be `Type.Tuple(Type.BoundAlias(x, Int), Type.BoundAlias(y, String))`?

Reply:
`<:<` is a type in the standard library, so it's not any special symbol.

BoundAlias is for the cases with the `as` keyword so it's a different thing. I reused TypedParam for named tuples already before, since it's good enough here.

Reply:
Anyway, I would leave it as is unless we want to introduce a special type for NamedTuples